Makefile

```diff
@@ -0,0 +1,124 @@
+artifact_name       := company-metrics-consumer
+version             := unversioned
+
+## Create help from comments in Makefile
+help:
+	@printf "%-20s %s\n" "Target" "Description"
+	@printf "%-20s %s\n" "------" "-----------"
+	@make -pqR : 2>/dev/null \
+        | awk -v RS= -F: '{if ($$1 !~ "^[#.]") {print $$1}}' \
+        | sort \
+        | egrep -v -e '^[^[:alnum:]]' -e '^$@$$' \
+        | xargs -I _ sh -c 'printf "%-20s " _; make _ -nB | (grep -i "^# Help:" || echo "") | tail -1 | sed "s/^# Help: //g"'
+
+.PHONY: all
+all:
+	@# Help: Calls methods required to build a locally runnable version, typically the build target
+	mvn clean install
+
+.PHONY: clean
+clean:
+	@# Help: Reset repo to pre-build state (i.e. a clean checkout state)
+	mvn clean 
+	rm -f ./$(artifact_name).jar
+	rm -f ./$(artifact_name)-*.zip
+	rm -rf ./build-*
+	rm -rf ./build.log-*
+
+.PHONY: build
+build:
+	@# Help: Pull down any dependencies and compile code into an executable if required
+	$(info Setting version: $(version))
+	mvn versions:set -DnewVersion=$(version) -DgenerateBackupPoms=false
+	$(info Packing version: $(version))
+	mvn package -Dmaven.test.skip=true
+	cp ./target/$(artifact_name)-$(version).jar ./$(artifact_name).jar
+
+.PHONY: test
+test: test-integration test-unit 
+	@# Help: Run all test-* targets (convenience method for developers)
+
+.PHONY: test-unit
+test-unit:
+	@# Help: Run unit tests
+	mvn test -Dskip.integration.tests=true
+
+.PHONY: test-integration
+test-integration:
+	@# Help: Run integration tests
+	mvn integration-test -Dskip.unit.tests=true
+
+,PHONY: run-local
```

Comment:
As identified in other projects this should be .PHONY and not ,PHONY.

Reply:
incorporated the change in file.